### DISCUSSION TITLE
The basic example from pyworkbench website does not work

### DISCUSSION THREAD

**Discussion by Juns-Cheng** [ansys/pyworkbench#176] [General]
Hello. I am using python 3.9.13, ANSYS 2023R2 on WIN 10. All I want to do is start a workbench session and run an existing wbjn by using pyansys. 
I follow the steps on the example web (https://examples.workbench.docs.pyansys.com/version/stable/examples/logging/main.html)
by typing the commands in python console. 
But when workbench starts, an error box pops up in ansys workbench:"'GuiOperation Processing commandline argument' threw exception during Invoke: CommandArgumentException: Unknown argument: EnvironmentPrefix" . 
And the python console says "ERROR:root:Failed to start Workbench service within reasonable timeout."
Did I miss any steps? Any suggestions?

**Comment by RobPasMue** [comment]
Transferring to PyWorkbench repository. Pinging @ansys/pyworkbench dev team

**Comment by vnamdeo** [comment]
@Juns-Cheng, 

The PyWorkbench library is compatible with Ansys v2024R2 and later versions (https://workbench.docs.pyansys.com/version/stable/getting-started.html). We kindly request that you utilize the most recent Ansys product releases. 

For the compatible python version to use, please refer to this documentation link (https://docs.pyansys.com/version/dev/supported_versions.html). 

Hope it helps!
